client/src/components/SortDirection/SortDirection.stories.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
```

Comment:
Reminder to other reviewers, `SortDirection` story was already reviewed and approved in #1079